stdlib/public/Platform/android.modulemap

```diff
@@ -0,0 +1,436 @@
+  export *
+}
+
+module _inet [system] {
+  // This module groups headers related to inet
+  // and networking.
+  header "ifaddrs.h"
+  header "netdb.h"
+  header "arpa/inet.h"
+  header "netinet/in.h"
+  header "netinet/in6.h"
+  export *
+}
+
+module _jni [system] {
+  header "jni.h"
+  export *
+}
+
+module _langinfo [system] {
+  header "langinfo.h"
+  export *
+}
+
+module _pthread [system] {
+  header "pthread.h"
+  export *
+}
+
+module _regex [system] {
+  header "regex.h"
+  export *
+}
+
+module _resolv [system] {
+  header "resolv.h"
+  export *
+}
+
+module _sched [system] {
+  header "sched.h"
+  export *
+}
+
+module _search [system] {
+  header "search.h"
+  export *
+}
+
+module _semaphore [system] {
+  header "semaphore.h"
+  export *
+}
+
+module _spawn [system] {
+  header "spawn.h"
+  export *
+}
+
+module _strings [system] {
+  header "strings.h"
+  export *
+}
+
+module _sys [system] {
+  header "syscall.h"
+  header "sysexits.h"
+  header "syslog.h"
+  export *
+}
+
+module _tar [system] {
+  header "tar.h"
+  export *
+}
+
+module _termio [system] {
+  header "termio.h"
+  header "termios.h"
+  export *
+}
+
+module _threads [system] {
+  header "threads.h"
+  header "bits/threads_inlines.h"
+  export *
+}
+
+module _uconfig_local [system] {
+  header "uconfig_local.h"
+  export *
+}
+
+module _ucontext [system] {
+  header "ucontext.h"
+  export *
+}
+
+module _wait [system] {
+  header "wait.h"
+  export *
+}
+
+module _wchar_uchar [system] {
+  header "uchar.h"
+  header "wchar.h"
+  header "bits/mbstate_t.h"
+  export *
+}
+
+module _xlocale [system] {
+  header "xlocale.h"
+  export *
+}
+
+// Additional modules in the 'android' subdirectory.
+module _android_defs [system] {
+  header "android/ndk-version.h"
+  header "android/versioning.h"
+  export *
+}
+
+module _android_apis [system] {
+  header "android/asset_manager_jni.h"
+  header "android/asset_manager.h"
+  header "android/log.h"
+  header "android/trace.h"
+  export *
+}
+
+// Additional modules in the 'bits' subdirectory.
+module _bits_io_inet [system] {
+  header "bits/in_addr.h"
+  header "bits/ioctl.h"
+  header "bits/ip_mreq_source.h"
+  header "bits/ip_msfilter.h"
+  header "bits/sa_family_t.h"
+  export *
+}
+module _bits_signaltypes [system] {
+  header "bits/signal_types.h"
+  export *
+}
+module _bits_stdatomic [system] {
+  header "bits/stdatomic.h"
+  export *
+}
+module _bits_timespec [system] {
+  header "bits/timespec.h"
+  export *
+}
+
+// Additional modules in the 'linux' subdirectory.
+module _linux_inet [system] {
+  header "linux/if.h"
+  export *
+}
+
+// Additional modules in the 'sys' subdirectory.
+module _sys_cdefs [system] {
+  header "sys/cdefs.h"
+  // Circular included header, so combine them
+  // into the same module.
+  header "android/api-level.h"
+  export *
+}
+module _sys_core [system] {
+  header "sys/endian.h"
+  header "sys/epoll.h"
+  header "sys/errno.h"
+  header "sys/eventfd.h"
+  header "sys/fcntl.h"
```

Comment:
Same for some of these other ones, it probably makes sense to put these with their regular headers i.e. with fcntl.h